upgrade/Stage_1.md

```diff
@@ -40,6 +40,18 @@ Note that the progress for the current stage will not show up in Argo before the
 
 For more information, see [Using the Argo UI](../operations/argo/Using_the_Argo_UI.md) and [Using Argo Workflows](../operations/argo/Using_Argo_Workflows.md).
 
+
+## In-family Upgrade requirement
```

Comment:
Update "in-family" to say a more recent version of the same release of CSM, e.g. CSM V1.3.0-rc.2 to CSM V1.3.0-rc.3

Reply:
Please check if the wording is good enough  @denniswalker